### DISCUSSION TITLE
transparency not working

### DISCUSSION THREAD

**Discussion by XxRiKxX** [tsujan/Kvantum#986] [Q&A]
on arch linux using hyprland. I tried sweet mars and fluent. both dont have transparency. using qt6ct as engine

**Comment by tsujan** [comment]
I'm afraid this place is Kvantum's bug tracker. I can't say anything about hyprland, qt6ct or themes that aren't included in Kvantum.

Converting your report to a Discussion; perhaps a user with one of those programs/themes could help you...

**Reply by XxRiKxX** [reply]
thanks I will wait then
